src/components/CtaSection/Cta.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../sass/abstracts/variables';
+
+.cta {
+  margin-top: 5rem;
+
+  &__image {
+    width: 100%;
+    margin-bottom: 2.5rem;
+  }
+
+  &__details {
+    align-items: center;
+    text-align: center;
+  }
+
+  &__btn {
+    margin-top: 1.625rem;
+  }
+
+  h2 {
```

Comment:
Yes it's better. By targetting <h2> and <p> inside of cta you don't target the global style of all h2.

Reply:
Okay. I actually thought that since the styling is component based, it will only affect the cta component. Did not think it will affect other parts when compiled.... interesting